packages/cli/src/utils/directory.test.ts

```diff
@@ -0,0 +1,84 @@
+import path from "path"
+import { coreCMSDir, coreDir, tmpCMSDir, tmpCMSWebhookUrlsFile, tmpCustomizationsSrcDir, tmpDir, tmpStoreConfigFile, tmpThemesCustomizationsFile, userCMSDir, userSrcDir, userStoreConfigFile, userThemesFileDir } from "./directory"
+
+const pathsToMatch = (expected: string, desired: string) => {
+  const expectedResolved = path.resolve(expected)
+  const desiredResolved = path.resolve(desired)
+
+  return expectedResolved === desiredResolved
+}
+
+describe('directory with base path', () => {
+  describe('coreDir', () => {
+    it('is the fastsstoreDir + core', () => {
+      expect(pathsToMatch(coreDir, './node_modules/@faststore/core')).toBe(true)
+    })
+  })
+
+  describe('tmpDir', () => {
+    it('is the cwd + .faststore', () => {
+      expect(pathsToMatch(tmpDir, './.faststore')).toBe(true)
+    })
+  })
+
+  describe('userDir', () => {
+    it('returns the directory of the starters package.json', () => {
```

Comment:
```suggestion
    it("returns the directory of the starter's package.json", () => {
```

Reply:
I was too lazy to do that. Thank you :pray: